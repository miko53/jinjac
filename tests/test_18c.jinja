{% for user in ussers %}-
This is user {{ user }}
{% endfor %}
dfdsg
{% for user in users %}-
This is user {{ usRr }}
{% endfor %}
dfdsg
