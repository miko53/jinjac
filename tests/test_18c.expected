
dfdsg
-
This is user 
-
This is user 
-
This is user 

dfdsg
